README.md

```diff
@@ -194,6 +200,9 @@ The PowerShell script implicitly creates a resource group by appending 'resource
 
 If you want to cleanup resources, delete that resource group in addition to any other resources you may have explicitly created as part of this tutorial.
 
+**[!NOTE]**
+> Azure data factory only support R4 version for now.

```

Comment:
To be precise here, "the Azure Data Factory tool" only supports R4 version for now. STU 3 support is planned.

Reply:
+1